proxy/hdrs/HdrTSOnly.cc

```diff
@@ -59,6 +59,11 @@ HTTPHdr::parse_req(HTTPParser *parser, IOBufferReader *r, int *bytes_used, bool
   ParseResult state = PARSE_RESULT_CONT;
   *bytes_used       = 0;
 
+  // FIXME: r maybe no has IObufferBlock object
```

Comment:
This isn't required because of the `r->block_read_avail()` a bit later. The code for that does the null check on the
block -
https://github.com/SolidWallOfCode/trafficserver/blob/master/iocore/eventsystem/P_IOBuffer.h#L493
Therefore it it returns a result > 0 then there is guaranteed to be a block.